DUnitX.InternalInterfaces.pas

```diff
@@ -80,6 +81,13 @@ interface
    // function AreEqual(const AFixtureResult : IFixtureResult) : boolean;
   end;
 
+  ITestDataProvider = interface
+    ['{1BFC9318-AF52-41BB-82D4-58166429885A}']
+    function GetCaseAmount(Methodname:string):integer;
```

Comment:
This should be GetCaseCount rather than Amount.

Reply:
No Problem :)